azure-pipelines.yml

```diff
@@ -64,7 +65,39 @@ jobs:
     setup:
       - bash: |
           sudo apt-get update &&
```

Comment:
Any chance to put the linux sqlite setup code into a own template, so we could remove the duplication here?

Reply:
Take a look at the install_rust template. Something similar should be done here.